docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,104 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+a number of {feature-vars} and a {depvar}, then making further predictions based 
```

Comment:
I think we need to introduce the concept of feature variables and dependent variables before we use them to define regression.

Reply:
I rephrased the first sentence to be more general because we introduce these concepts in the Feature variables section. The aim of this introductory section is to provide a basic understanding of the analysis, so I think the deeper terminology could be explained later.